src/reducers/user.reducer.js

```diff
@@ -6,12 +6,11 @@ import {
   SET_AUTH_ERROR,
   RESET_AUTH_ERROR,
   SET_USER_IN_STATE,
-  FETCH_USER_REPOSITORIES,
   SET_USER_REPOSITORIES
 } from 'actions/user.actions';
 
 const initialState = {
-  token: null,
+  token: false,
```

Comment:
why did you change the default value to false? it makes much more sense as null (which indicates you don't have one)

Reply:
There is a play between null, false and true here, because i do not render the app until i get null or true so the PrivateRoute component won't redirect me to Login page before i get any value. set it to null and you will never be able to access any page.